examples/TID_model_custom_property/Modelling_TID_TNID_example_notebook.ipynb

```diff
@@ -0,0 +1,471 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    def generate_lognormal_pdf_and_cdf(x_mean, x_sigma, n_samples):
</pre><p>Can you elaborate a bit why you use a lognormal distribution? </p><p> </p><p>Also, why not use use KDE to estimate the PDF / CDF? (see here https://scikit-learn.org/stable/modules/density.html#kernel-density-estimation ) That would just be one line. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aidotse/PASEOS/pull/168/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p>In the papers above they mention the H(x) is integral(lognormal). g(x) depends on the device. lognormal is also good for some devices</p>